modules/cnf-topology-aware-lifecycle-manager-PAO-update.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+// Epic CNF-4767 (4.11), Story TELCODOCS-593
+// * scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="talo-PAO-update_{context}"]
+= Removing Performance Addon Operator subscriptions from deployed clusters
+
+In earlier versions of {product-title}, the Performance Addon Operator was used to implement automatic tuning to achieve low latency performance for OpenShift applications. In {product-title} 4.11, these functions are part of the Node Tuning Operator. 
+
+Do not install the Performance Addon Operator on clusters running {product-title} 4.11 or later. When you upgrade to {product-title} 4.11,  the Node Tuning Operator automatically removes the Performance Addon Operator. However, you  need to manually remove any Performance Addon Operator subscriptions to prevent a reinstallation of the operator. The reference DU profile includes the Performance Addon Operator in the `common-ranGen.yaml` `PolicyGenTemplate`. To remove the subscription from deployed spoke clusters you must update this policy.
+
+.Prerequisites
+
+* Create a Git repository where you manage your custom site configuration data.
```

Comment:
Shouldn't the repo be accessible from the hub cluster so it can work with ArgoCD? If yes, you can find an example sentence here: http://file.emea.redhat.com/rohennes/conPR-telcodocs-593/scalability_and_performance/ztp-deploying-disconnected.html#ztp-using-pgt-to-update-source-crs_ztp-deploying-disconnected